cfme/scripting/quickstart.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import print_function
+
+import os
+import sys
+import argparse
+import subprocess
+import hashlib
+
+IS_SCRIPT = sys.argv[0] == __file__
+CWD = os.getcwd()  # we expect to be in the workdir
+REDHAT_BASED = os.path.isfile('/etc/redhat-release')
+CREATED = object()
+REQUIREMENT_FILE = 'requirements/frozen.txt'
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--mk_virtualenv", default="../cfme_venv")
+parser.add_argument("--system-site-packages", action="store_true")
+parser.add_argument("--config_path", default="../cfme-qe-yamls/complete/")
+
+
+def call_or_exit(command, shell=False):
+    try:
+        res = subprocess.call(command, shell=shell)
```

Comment:
Maybe add `print('Executing: {}'.format(' '.join(command)))` so the person who executes it sees exactly what happens.